consensus/state.go

```diff
@@ -1833,9 +1830,14 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 		}
 	}
 
+	blockSize := block.Size()
+
+	// trace some metadata about the block
+	schema.WriteBlock(cs.traceClient, block, blockSize)
```

Comment:
[non-blocking]
Why are we passing the block and also the size? isn't the size already inside?

Reply:
good question, taking the size requires serializing the entire block, so I wanted to avoid doing that twice since we're already doing it